reference/conanfile/tools/cmake.rst

```diff
@@ -137,6 +137,42 @@ Use the **build_context_build_modules** attribute to specify require names to in
         cmake.generate()
 
 
+Properties
```

Comment:
This was refactored, and now lives in the ``cmake/cmakedeps.rst`` file. Maybe it has not been merged back to develop?